docs/best-practices/gpu-and-rdma-joint-allocation.md

```diff
@@ -0,0 +1,1058 @@
++-----------------------------------------+------------------------+----------------------+
+|   2  Tesla P40                      Off |   00000000:82:00.0 Off |                    0 |
+| N/A   37C    P8             10W /  250W |       0MiB /  23040MiB |      0%      Default |
+|                                         |                        |                  N/A |
++-----------------------------------------+------------------------+----------------------+
+|   3  Tesla P40                      Off |   00000000:83:00.0 Off |                    0 |
+| N/A   30C    P8             10W /  250W |       0MiB /  23040MiB |      0%      Default |
+|                                         |                        |                  N/A |
++-----------------------------------------+------------------------+----------------------+
+
++-----------------------------------------------------------------------------------------+
+| Processes:                                                                              |
+|  GPU   GI   CI        PID   Type   Process name                              GPU Memory |
+|        ID   ID                                                               Usage      |
+|=========================================================================================|
+|  No running processes found                                                             |
++-----------------------------------------------------------------------------------------+
+```
+
+In addition to GPU, we also need to setup rdma environment on the worker nodes in advance.
+
+- Plan the physical NIC for the test
+
+  | node name | nic name                   | Nic model                                                    | NAD name                                        | Ip address   | remark                                                       |
+            | --------- | -------------------------- | ------------------------------------------------------------ | ----------------------------------------------- | ------------ | ------------------------------------------------------------ |
+  | K8s-node1 | ens11f0np0<br />ens11f1np1 | 01:00.0 Ethernet controller [0200]: Mellanox Technologies MT27800 Family [ConnectX-5] [15b3:1017]<br/>01:00.1 Ethernet controller [0200]: Mellanox Technologies MT27800 Family [ConnectX-5] [15b3:1017] | sriov-attach-k8s-node1-ens11f0np0-kubeflow-conf | 10.20.12.121 | To simplify the testing, we create pod01 and have it schedule directionally to node1 and occupy the VF on node1 |
+  | K8s-node2 | ens3f0np0<br />ens3f1np1   | 81:00.0 Ethernet controller [0200]: Mellanox Technologies MT27800 Family [ConnectX-5] [15b3:1017]<br/>81:00.1 Ethernet controller [0200]: Mellanox Technologies MT27800 Family [ConnectX-5] [15b3:1017] | sriov-attach-k8s-node2-ens3f0np0-kubeflow-conf  | 10.20.12.134 | To simplify the testing, we create pod02 and have it schedule directionally to node2 and occupy this VF |
```

Comment:
Could the information of physical NIC here be structured inside a code block or something?